### PR TITLE
(DOCSP-18481): Scaffold new schemas & rules sections

### PR DESCRIPTION
**Pull request by nlarew**
### Jira

- (DOCSP-18481): Scaffold new schemas & rules sections

### Staged Changes (Requires MongoDB Corp SSO)

- [Schemas](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/schema-rules/schemas)
- [Rules](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/schema-rules/rules)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/cloud.txt [hunk 1]
```diff
@@ -8,19 +8,21 @@ MongoDB Realm Application Services
    :titlesonly:
    :hidden:
    
+   Realm Sync </sync>
+   Schemas </sync>
```
Comment (dacharyc) [inline source/cloud.txt:12]: Is it intentional that all three of these are going to `/sync`? It's unexpected to me when I'm clicking around the ToC.

**Review by dacharyc — COMMENTED** [review]
Looks like the two new sections, Schemas and Rules, are orphaned, and the ToC has them all going to `/sync`. Typo or intentional?

File: source/cloud.txt (hunk 1 above)
Comment (nlarew) [reply source/cloud.txt:12]: Oops good catch! Updated!

**Review by dacharyc — APPROVED** [review]
LGTM!

**Pull request merged**